webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
+var SRC_DIR = path.join(__dirname, '/client/src');
+var DIST_DIR = path.join(__dirname, '/client/public');
+
+module.exports = {
+  entry: "./server/index.js",
+  output: {
+    filename: "bundle.js",
+    path: path.resolve(__dirname, "public")
+  },
+  module: {
+    rules: [
+      {
+        test: /\.m?js$/,
+        exclude: /(node_modules|bower_components)/,
+        use: {
+          loader: "babel-loader",
+          options: {
+            presets: ["@babel/preset-env"]
+          }
+        }
+      }
+    ]
+  }
+};
```

Comment:
As Kimmy pointed out to me, we likely won't need webpack on our proxy unless we get deep into advanced content. The way I understand it, we're going to be generating our bundles in their respective repos using a build script, and then the proxy will just tap that build .js file and render that way. I think you can freely nix this file for the time being.